.eslintrc.json

```diff
@@ -7,6 +7,14 @@
     "jest": true,
     "es6": true
   },
+  "extends": ["next/core-web-vitals", "plugin:prettier/recommended"],
+  "rules": {
+    "prettier/prettier": ["error", {
```

Comment:
we have a prettier file close to the eslint.json 
`.prettierrc`, can you please move all the settings from it here and remove that file?